Jenkinsfile

```diff
@@ -51,6 +51,8 @@ pipeline {
     stage('Checkout') {
       options { skipDefaultCheckout() }
       steps {
+        // For testing purposes, let's print the env variable CI
```

Comment:
@v1v Revert these?

Reply:
yes please

Reply:
Done. I'll leave the CI print in the mage dump since that will potentially be useful in the future given we are now depending on that in the mage script.